lib/riax_vnode.ex

```diff
@@ -1,69 +1,136 @@
 defmodule Riax.VNode do
+  @moduledoc """
+  This module is an implementation of the VNode
```

Comment:
I think the current documentation is ok but you should add some general information about this module (like what is a vnode) so the user of the library knows what is she/he implementing.